app/src/main/java/com/github/leuludyha/ibdb/presentation/components/books/reading_list/controls/ReadingStateControlViewModel.kt

```diff
@@ -1,13 +1,40 @@
 package com.github.leuludyha.ibdb.presentation.components.books.reading_list.controls
```

Comment:
Can we just group all actions (save/load/delete) in a separate viewModel or useCase or whatever that we can then reuse with @Inject in other viewModels ?

Reply:
Sure, you think having a `LocalUseCases` class grouping all of them would be better?

Reply:
Maybe something like `LocalRepository` which holds the load, delete and save method would be more suitable but yes, the idea is to regroup them

Reply:
Then why not passing directly the `LibraryRepository`? How I see things is: either you want to be clear and restrictive on what functionalities you're using and use the `UseCase`s, or you want to be general and have access to a lot of methods and use `LibraryRepository`.
I agree that conceptually it is better to restrain our api to the necessary (hence having only the local functions of the Repository) but then I think we can simply use the `UseCase`s

Reply:
Right, I was confused then, nevermind